webapp/app.py

```diff
@@ -210,7 +217,7 @@ def job_details(job_id, job_title):
     return flask.render_template("/careers/job-detail.html", **context)
 
 
-@app.route("/careers/start")
+@app.route("/careers/career-explorer")
```

Comment:
There are links to the old page on the homepage and navigation which leads to a 404. Can you add a redirect and update links across this project.